packages/merge-styles/src/index.ts

```diff
@@ -18,7 +18,7 @@ export { fontFace } from './fontFace';
 
 export { keyframes } from './keyframes';
 
-export { IStyleSheetConfig, ICSPSettings, InjectionMode, Stylesheet } from './Stylesheet';
+export * from './Stylesheet';
```

Comment:
any reason for this?

Reply:
When importing v7's utilities into v0, when building `@fluentui/docs` everything was being parsed. v0 uses the babel compiler, while v7 uses the typescript compiler. v0's parsing was complaining because `ICSPSettings` and `IStyleSheetConfig` are interfaces that it was throwing out and not finding when exported here. Exporting as `*` instead of individually fixes this issue.

There were some concerns about doing this and there is a 100+ long thread offline under the `Engineering Systems` teams channel. The concerns were mostly related to webpack incorrectly tree shaking `export *` when it's two levels deep. However, I checked that this wasn't happening by creating a new react app and consuming both the regular version of `office-ui-fabric-react` and a tarball with the changes without much change in the file size at all.

Regular without changes:

```
File sizes after gzip:
  86.85 KB  build\static\js\2.a2312137.chunk.js
  772 B     build\static\js\runtime-main.a43b6eb2.js
  556 B     build\static\css\main.d1b05096.chunk.css
  541 B     build\static\js\main.87aabca2.chunk.js
```

Tarball with changes:

```
  86.87 KB (+15 B)  build\static\js\2.5b88c310.chunk.js
  772 B             build\static\js\runtime-main.a43b6eb2.js
  556 B             build\static\css\main.d1b05096.chunk.css
  541 B             build\static\js\main.87aabca2.chunk.js
```

Given this I think it would be safe to include this change.